lib/valid_email2.rb

```diff
@@ -1,24 +1,34 @@
 # frozen_string_literal: true
 
 require "valid_email2/email_validator"
+require_relative "./helpers/deprecation_helper"
 
 module ValidEmail2
+  
   BLACKLIST_FILE  = "config/blacklisted_email_domains.yml"
+  DENY_LIST_FILE  = "config/deny_listed_email_domains.yml"
   WHITELIST_FILE  = "config/whitelisted_email_domains.yml"
+  ALLOW_LIST_FILE  = "config/allow_listed_email_domains.yml"
   DISPOSABLE_FILE = File.expand_path('../config/disposable_email_domains.txt', __dir__)
 
   class << self
+    extend DeprecationHelper
+
     def disposable_emails
       @disposable_emails ||= load_file(DISPOSABLE_FILE)
     end
 
     def blacklist
-      @blacklist ||= load_if_exists(BLACKLIST_FILE)
+      @deny_list ||= load_if_exists(DENY_LIST_FILE || BLACKLIST_FILE)
     end
+    alias :deny_list :blacklist
+    deprecate_method :blacklist, :deny_list
```

Comment:
I considered using `Gem::Deprecate`, but since I wouldn't have been able to make use of it in `EmailValidator` since we're deprecating options and not methods, I ultimately decided to leave it out.